proto-build/buf.sdk.gen.yaml

```diff
@@ -3,7 +3,12 @@ plugins:
   - plugin: buf.build/community/neoeinstein-prost:v0.3.1
     out: .
     opt:
+      - compile_well_known_types
+#      - extern_path=.google.protobuf=::pbjson_types
```

Comment:
Looks like this can be removed

Reply:
So I guess the problem was I added the config to `tonic` instead of `prost`?

Reply:
I suppose so?

Reply:
I can fix it up in a followup